packages/terra-show-hide/src/terra-dev-site/test/show-hide/StatesShowHide.test.jsx

```diff
@@ -1,30 +0,0 @@
-import React from 'react';
-import ShowHide from '../../../ShowHide';
-
-const StatesShowHideTest = () => (
```

Comment:
This file is no longer needed, the focus, hover, and active states are tested by hitting the default test example and using the click events, keyEvents, and mouse emulation in the wdio test to trigger the various states.